### DISCUSSION TITLE
希望在拓展倍速中加一个根据up主记忆倍速的功能

### DISCUSSION THREAD

**Discussion by 574094130** [the1812/Bilibili-Evolved#3738] [功能建议 / Ideas]
如题，一般视频只会看一次，分视频记忆感觉用处不大，全局又会根据不同的up每次选择的倍速不一样需要分别调整有些麻烦。希望能识别UP主，分别记忆视频倍速

**Comment by timongh** [comment]
这个可以，再联想一下的话，还可以根据标题规则，根据分P，根据合集等等来倍速。

再联想一下，一些其他设置，比如初始音量，连播设置也可以按这个来。

。。。我只是联想到了，随口说说（逃

**Reply by Ethkuil** [reply]
发明一个 DSL(
